packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -147,7 +147,7 @@ export default async args => {
             },
             treeshake: { moduleSideEffects: false },
             plugins: [
-                url(),
+                url({limit: 100000}),
```

Comment:
Changes in the piw-tools requires a bump and release in npmjs

Reply:
Annoying question: Why is this necessary ?

Reply:
@diego-antonelli Allright, not sure how to create a release though..
@mmehmetAliIzci Because the default file limit is 14 kb

Reply:
```suggestion
                url({ include: ["**/*.svg"], limit: 102400 }), // SVG file size limit of 100 kB,
```